test/config.yaml

```diff
@@ -1,4 +0,0 @@
-sampletable: pasilla_sampletable.tsv
-sample_dir: pasilla
-assembly: dm6
-data_dir: /data/LCDB/references
```

Comment:
Do you want me to move thing and then do the cluster-config merge?
I am running the cluster test now.

On Thu, Jun 30, 2016 at 9:14 AM, Ryan Dale <notifications@github.com> wrote:

> In test/config.yaml
> <https://github.com/lcdb/lcdb-workflows/pull/38#discussion_r69128877>:
>
> > @@ -1,4 +0,0 @@
> > -sampletable: pasilla_sampletable.tsv
> > -sample_dir: pasilla
> > -assembly: dm6
> > -data_dir: /data/LCDB/references
>
> sure, how about something like
>
> resources/
>   requirements.txt
>   test_data/
>     pasilla/
>       get-data.sh
>       sampletable.tsv
>       config.yaml
>       adapters.fa
>     splicing/
>       get-data.sh
>       sampletable.tsv
>       config.yaml
>
> and move the existing schemas over to lcdb maybe? Then I could build some
> tests for the schema config generation on top of what you currently have in
> that dir.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/lcdb/lcdb-workflows/pull/38/files/f9639e14a375f6b78c75a077b54fa74a431aa4a7#r69128877>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABLtcOAyJ3-hDQMKaRxXMeaQW0GIQOMpks5qQ8EjgaJpZM4JBIcI>
> .
>